### ISSUE TITLE
Passing string values in durationObject multiplies it on 10.

### ISSUE THREAD

**Issue by franzzua** [moment/luxon#402]
line 3 should be same as line 2 or throw error
```
const {DateTime} = require('luxon');

const dateTime = DateTime.utc(2019,1,1);

console.log(dateTime.toISO());                            // 2019-01-01T00:00:00.000Z
console.log(dateTime.plus({ minutes: 1 }).toISO());       // 2019-01-01T00:01:00.000Z
console.log(dateTime.plus({ minutes: "1" }).toISO());     // 2019-01-01T00:10:00.000Z
```
Same thing with { days: "1" } or { seconds: "1" }

**Comment by ricardobanegas** [comment]
Noticed bug also, I took care of it by parsing value to int.

**Comment by icambron** [comment]
I'd take a PR that throws on a non-numeric value. I bet it goes somewhere in this Utils.normalizeObject: https://github.com/moment/luxon/blob/master/src/impl/util.js#L192

**Comment by icambron** [comment]
Fixed in #421
